Upstream-Status: Pending

this patch avoids these issues with automake 1.12

| configure.ac:304: error: automatic de-ANSI-fication support has been removed
| tests/Makefile.am:12: error: automatic de-ANSI-fication support has been removed

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@.intel.com>
2012/05/02

Index: mpfr-3.1.0/configure.ac
===================================================================
--- mpfr-3.1.0.orig/configure.ac
+++ mpfr-3.1.0/configure.ac
@@ -301,7 +301,6 @@ if test "$test_CFLAGS" != set && test -n
 fi
 
 AM_PROG_CC_C_O
-AM_C_PROTOTYPES
 
 case $host in
   *-apple-darwin*)
Index: mpfr-3.1.0/tests/Makefile.am
===================================================================
--- mpfr-3.1.0.orig/tests/Makefile.am
+++ mpfr-3.1.0/tests/Makefile.am
@@ -9,7 +9,7 @@
 # PARTICULAR PURPOSE.
 
 
-AUTOMAKE_OPTIONS = 1.6 gnu $(top_builddir)/ansi2knr
+AUTOMAKE_OPTIONS = 1.6 gnu
 
 # tversion is run
 #   * at the beginning so that the user gets a possible version error
